ci/run.bat

```diff
@@ -0,0 +1,4 @@
+call dub test --skip-registry=all --compiler=%DC%
```

Comment:
Why are these `call` commands instead of regular commands?

Reply:
The Microsoft help for the CALL command rather misleadingly states "Calls one batch program from another without stopping the parent batch program" it is true that the parent does not STOP, but it does PAUSE while the second script runs.

Reply:
But there's a little bit of improvement here.
Whether the ERROR LEVEL is output correctly.

Reply:
I think these calls should not be asynchronous though. So it is right that each command should run one at a time. So the `call` is unnecessary I think..